package-lock.json

```diff
@@ -1,20 +1,22 @@
 {
   "name": "schoolbox-api",
-  "version": "1.2.6",
+  "version": "1.2.8",
   "lockfileVersion": 2,
   "requires": true,
   "packages": {
     "": {
       "name": "schoolbox-api",
-      "version": "1.2.6",
+      "version": "1.2.8",
       "dependencies": {
-        "@redocly/openapi-cli": "^1.0.0-beta.50"
+        "@redocly/cli": "^1.0.0-beta.109",
```

Comment:
Are we even still using redocly? Haven't we moved to rapidoc

Reply:
It's still needed for actually compiling the multiple spec fragments into a single spec.